MontyHallv2/DoorCreation/Door.cs

```diff
@@ -0,0 +1,42 @@
+namespace MontyHallv2.DoorCreation;
+
+public class Door
+{
+    private bool _opened;
+    private bool _hasCar;
+    private bool _playerPicked;
+
+    public void OpeningDoor()
+    {
+        _opened = true;
+    }
+
+    public bool HasPlayerPicked()
+    {
+        return _playerPicked;
+    }
+
+    public void PlayerPickedDoor()
+    {
+        _playerPicked = true;
+    }
+
+    public void PlayerUnpickDoor()
+    {
+        _playerPicked = false;
+    }
+    public void InjectCarToDoor()
+    {
+        _hasCar = true;
+    }
+
+    public bool HasCar()
+    {
+        return _hasCar;
+    }
+
+    public bool IsDoorOpened()
```

Comment:
```suggestion
    public bool IsOpen()
```
we're in the door class. reads nicer `_door.IsOpen()` vs `_door.IsDoorOpened()`